kernel/kernel_config-4.9.x

```diff
@@ -243,16 +243,13 @@ CONFIG_OPROFILE=y
 # CONFIG_OPROFILE_EVENT_MULTIPLEX is not set
 CONFIG_HAVE_OPROFILE=y
 CONFIG_OPROFILE_NMI_TIMER=y
-CONFIG_KPROBES=y
+# CONFIG_KPROBES is not set
```

Comment:
I think it would probably block the kprobe instrumenting parts of perf, but not the events and such (that's under a different flag). I can drop it from the series if you like.